transport/ice/IceSession.cpp

```diff
@@ -1456,13 +1457,13 @@ void IceSession::CandidatePair::processTimeout(const uint64_t now)
     if (state == InProgress)
     {
         if (remoteCandidate.type == IceCandidate::Type::HOST &&
-            now - startTime > _config.hostProbeTimeout * utils::Time::ms)
+            utils::Time::diffGT(startTime, now, _config.hostProbeTimeout * utils::Time::ms))
```

Comment:
This could happen due to ICE request with prflx candidate arriving. Timestamp is then from transport which may be timstamped before the HOST probe started from SMB. The probe would fail immediately and the entire ice session failed since there are no other canddiates yet.